.NET/Microsoft.Recognizers.Text.DateTime/Parsers/CJK/BaseCJKDatePeriodParser.cs

```diff
@@ -2692,21 +2713,21 @@ private DateTimeResolutionResult ParseDecade(string text, DateObject referenceDa
                 }
 
                 beginLuisStr = DateTimeFormatUtil.LuisDate(-1, 1, 1);
-                beginLuisStr = beginLuisStr.Replace("XXXX", beginYearStr);
+                beginLuisStr = beginLuisStr.Replace(Constants.TimexFuzzyYear, beginYearStr);
```

Comment:
All this string processing should be common in timex generation. Also, beginLuisStr and endLuisStr don't need to mention LUIS.

Reply:
done